unison-src/builtin-tests/interpreter-tests.md

```diff
@@ -2,6 +2,7 @@
 Note: This should be forked off of the codebase created by base.md
 
 ```ucm
+.> builtins.merge
```

Comment:
No need to do this. The transcript is already pulling base, which includes names for all builtins.